doc/classes/CubeMap.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CubeMap" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		A CubeMap is a 6 sided 3D texture.
 	</brief_description>
 	<description>
+		A CubeMap is a 6 sided 3D texture typically used for faking reflections. This usually delivers the best performace to make an object look as if it is reflecting it's surroundings.
```

Comment:
performance, its

Reply:
Thanks, it was a little difficult to read. I changed it to "[...] performance, it makes [...]".